src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Transforms;
+using System.Collections.Generic;
```

Comment:
do you even need changes in this file?

Reply:
No, I will revert this.

Reply:
Did you?

---
In reply to: [243129553](https://github.com/dotnet/machinelearning/pull/1710#discussion_r243129553) [](ancestors = 243129553)

Reply:
you got me. I am doing it now 


---
In reply to: [243448988](https://github.com/dotnet/machinelearning/pull/1710#discussion_r243448988) [](ancestors = 243448988,243129553)